src/styles/index.scss

```diff
@@ -1,3 +1,49 @@
+@import 'https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap';
+@import 'card';
+@import 'stars';
+
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
+  padding: 0;
+}
+
+.stars {
+  // .stars__star-block
+
+  &__star-block {
+    display: flex;
+  }
+
+  &__reviews {
+    margin: 0;
+    color: #060b35;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 1.4;
+  }
+
+  // .stars__star
+
+  &__star {
+    background-image: url(/src/images/star.svg);
```

Comment:
The `background-image` URL should be relative to the `index.scss` file. Ensure the path is correct so that the image can be loaded properly.